### PR TITLE
gh-102558: Fix AttributeError or repr() of enum member

### PR DESCRIPTION
**Pull request by corona10**
* Issue: gh-102558

### CODE REVIEW

File: Lib/test/test_enum.py [hunk 1]
```diff
@@ -2893,6 +2893,14 @@ def __new__(cls, c):
         self.assertEqual(FlagFromChar.a, 158456325028528675187087900672)
         self.assertEqual(FlagFromChar.a|1, 158456325028528675187087900673)
 
+    def test_multiple_superclasses(self):
+        class _EnumSuperClass(metaclass=EnumMeta):
+            pass
+        class E(_EnumSuperClass, Enum):
+            A=1
+        self.assertEqual(format(E.A), "E.A")
```
Comment (vashek) [inline Lib/test/test_enum.py]: I think the test should call repr(), not format()
```suggestion
        self.assertEqual(repr(E.A), "<E.A: 1>")
```

Comment (corona10) [reply Lib/test/test_enum.py]: Thanks! It was my mistake.

File: Lib/test/test_enum.py [hunk 2]
```diff
@@ -2893,6 +2893,14 @@ def __new__(cls, c):
         self.assertEqual(FlagFromChar.a, 158456325028528675187087900672)
         self.assertEqual(FlagFromChar.a|1, 158456325028528675187087900673)
 
+    def test_multiple_superclasses(self):
```
Comment (sobolevn) [inline Lib/test/test_enum.py]: We have tests for `repr` defined here: https://github.com/python/cpython/blob/ffa20484dfcd3e542d3fb76137160abc2a1f98a0/Lib/test/test_enum.py#L612

Let's move it closer maybe?

File: Lib/test/test_enum.py [hunk 3]
```diff
@@ -2893,6 +2893,14 @@ def __new__(cls, c):
...
+        class _EnumSuperClass(metaclass=EnumMeta):
+            pass
+        class E(_EnumSuperClass, Enum):
+            A=1
```
Comment (sobolevn) [inline Lib/test/test_enum.py]:
```suggestion
            A = 1
```

nit :)

**Comment by vashek** [comment]
Is there anything that should be done to get this reviewed and merged, or are we just waiting for @ethanfurman's attention and time? Sorry, not meaning to nag, just unfamiliar with the process here. Thanks.

**Review by ethanfurman — APPROVED** [review]

**Pull request merged**